### DISCUSSION TITLE
How to hide "sh" title/label

### DISCUSSION THREAD

**Discussion by geril07** [ibhagwan/fzf-lua#1635] [Q&A]
How can I hide shell title("sh")
![image](https://github.com/user-attachments/assets/e0606cde-5908-49a3-8b16-c8c77b097e24)

**Comment by ibhagwan** [comment]
This is rather odd and shouldn’t appear there, what’s you fzf version / `$FZF_DEFAULT_OPTS`?

What’s your config? Does this happen with mini.sh?

**Comment by geril07** [comment]
Oh, I found that the title was from a winbar, thanks!

**Comment by geril07** [comment]
@ibhagwan Btw, is prompt buf filetype empty? Maybe it's a good idea to set it to something like `FzfLuaPrompt`. As I know [telescope](https://github.com/nvim-telescope/telescope.nvim/blob/2eca9ba22002184ac05eddbe47a7fe2d5a384dfc/lua/telescope/pickers.lua#L615-L617) have `TelescopePrompt` filetype. For me it's not required

**Reply by ibhagwan** [reply]
> @ibhagwan Btw, is prompt buf filetype empty? Maybe it's a good idea to set it to something like `FzfLuaPrompt`. As I know [telescope](https://github.com/nvim-telescope/telescope.nvim/blob/2eca9ba22002184ac05eddbe47a7fe2d5a384dfc/lua/telescope/pickers.lua#L615-L617) have `TelescopePrompt` filetype. For me it's not required

Main buffer type is `fzf`.
